src/Controller/EnMarche/DonationController.php

```diff
@@ -53,20 +53,26 @@ public function informationsAction(Request $request)
             return $this->redirectToRoute('donation_index');
         }
 
-        $donationRequest = $this->get(DonationRequestUtils::class)
-            ->createFromRequest($request, (float) $amount, $subscription, $this->getUser())
-        ;
+        $donationRequest = $donationRequestUtils->createFromRequest(
+            $request,
+            (float) $amount,
+            $subscription,
+            $this->getUser()
+        );
 
         $form = $this->createForm(DonationRequestType::class, $donationRequest, ['locale' => $request->getLocale()]);
 
         if ($form->handleRequest($request)->isSubmitted() && $form->isValid()) {
             $this->get('app.donation_request.handler')->handle($donationRequest);
+            $donationRequestUtils->resetDonationRequest();
```

Comment:
From this context it would make more sense to call the method `terminateDonationRequest()`